twap/Makefile

```diff
@@ -0,0 +1,52 @@
+prev_ta := $(shell jq '.function_calls[0].transitive_attestation' tmp/prev.json )
+prev_ea := $(shell jq '.enclave_attested_application_public_key.enclave_attestation' tmp/prev.json | sed 's/\//\\\//g' )
```

Comment:
When testing against the beta.3 server, I realized that the EA contains forward slashes that were messing with sed syntax during `VAR_EATTEST` replacements. To avoid that, I'm escaping the forward slashes here

Reply:
I tried `-r` to make sure, but that's not the problem. The EA contains strings like `t1b/NhX4`. The forward slash messes with the `sed` replacement strings in the `iteration` and `twap` targets.

I'm going to move this forward slash replacement down into those targets, since that's where its needed, so that `prev_ea` is not modified here for a relatively far away reason.